_data/AnirudhMukherjee/AnirudhMukherjee.md

```diff
@@ -0,0 +1,5 @@
+name: AnirudhMk
+institution: Dwarkadas J. Sanghvi College of Engineering
+profile_pic: ani.jpg

```

Comment:
Hey @AnirudhMukherjee , please make `jpg` to `JPG`, to make it exact as per the file extension.

Reply:
On it